docs/docs/prototype-an-assistant.mdx

```diff
@@ -4,7 +4,7 @@ sidebar_label: 'Prototype an Assistant'
 title: 'Prototype an Assistant'
 ---
 import useBaseUrl from '@docusaurus/useBaseUrl';
-import Prototyper, { Chat, DownloadButton, TrainButton } from '@site/src/components/prototyper';
+import Prototyper, { DownloadButton, TrainButton } from '@theme/Prototyper';
```

Comment:
just to be sure I understand: why is it under the `themes/` folder instead of the `components/` folder?

Reply:
I can't quite remember now. It seemed like the `@theme` alias was more stable than `@site`; but anyway I found it clearer to have customized and overridden components in one spot

Reply:
it makes contributions and later updates to the theme more complicated for developers who do not know how docusaurus work, IMO. But not blocking, we can always move it back if we need to.